### DISCUSSION TITLE
What methods can I use to work out if its me or (some) Mastodon?

### DISCUSSION THREAD

**Discussion by lordmatt** [Automattic/wordpress-activitypub#224] [Q&A]
I've set up ActivityPub on one of my blogs. On phpc.social I can follow myself on the blog. Yet on mastodon.social and mastodonapp.uk boosts are hidden and they both find nothing to subscribe to.

What I am trying to figure out is if it is a Mastodon issue, a me issue, or something else (say a bug or malformed JSON or something). What steps can I take to work out why my presence only exists for some instances?

**Comment by pfefferle** [comment]
Have you checked the site health page in your WordPress instance? Any errors there?

Please let me know your WebFinger resource (username@domain.tld), so I could try to help debugging.

**Comment by lordmatt** [comment]
lordmatt@lordmatt.co.uk is the WebFinger resource which webfinger.net produces a decent-looking output for and it is found correctly by phpc.social but not mastodon.social or mastodonapp.uk.

**Comment by pfefferle** [comment]
It seems to be a problem with your SSL certificate or the validation of that cerificate on Mastodon side. If I try to follow your blog I get an error "503 remote SSL certificate could not be verified": https://github.com/pfefferle/wordpress-activitypub/issues/11

The issue on Mastodon side: https://github.com/mastodon/mastodon/issues/7832
